### DISCUSSION TITLE
Modifying the UI and Updating the Collection

### DISCUSSION THREAD

**Discussion by hattorihanzo86** [pocketbase/pocketbase#2989] [Q&A]
Hi i added a function button in bulk dialog   i added button Approved Reject 
im planning to use this function to update collection with has table Status
how can i achieved this?

```
   function toggleSelectRecord(record) {
        if (!bulkSelected[record.id]) {
            bulkSelected[record.id] = record;
        } else {
            delete bulkSelected[record.id];
        }

        bulkSelected = bulkSelected; // trigger reactivity
    }
```

**Comment by ganigeorgiev** [comment]
The web APIs doesn't support bulk operations at the moment, if that's what you are asking (_eventually will be handled as part of #48_).

You'll have to send individual requests for each record OR [register a custom Go route](https://pocketbase.io/docs/router/#register-new-route) but there are not enough details to understand the requirements to help you with the latter.

**Reply by hattorihanzo86** [reply]
Ah so the delete from collection only works it does not support all id that selected can update a collection ?

**Reply by ganigeorgiev** [reply]
You can delete only a single record per request at the moment (_unless you register a custom Go route_).

**Reply by hattorihanzo86** [reply]
is there a way to make a go route like Update(CollectionName,id,newValue) if it's single per record request can you guide me i want to give a try.

**Reply by ganigeorgiev** [reply]
I'm not sure I follow.
If you want to update only a single record per request why don't use the `update()` method of the JS SDK:
```js
await pb.collection("example").update(id, { yourField: newValue })
```
?

**Reply by hattorihanzo86** [reply]
Im trying to experiment via RecordsList.svelte

**Reply by ganigeorgiev** [reply]
I'm confused. Do you mean [`RecordsList.svelte`](https://github.com/pocketbase/pocketbase/blob/master/ui/src/components/records/RecordsList.svelte)? If Yes, please note that **the Admin UI is not extendable** at the moment.

**Reply by hattorihanzo86** [reply]
Ah ic yes the Admin UI understand hopefully we can extend this in your future release.
